instance/migrations/0074_openstack_region.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.3 on 2017-02-12 13:10
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('instance', '0073_openstack_settings_data_defaults'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='openedxinstance',
+            name='openstack_region',
+            field=models.CharField(default=settings.OPENSTACK_REGION, max_length=16),
```

Comment:
@pomegranited The important bit is that you need to pass the `preserve_default=False` parameter to `AddField` to tell Django that the default value is only stored for the purpose of this migration.